src/phoebus_utils/preprocessor_utils.hpp

```diff
@@ -0,0 +1,7 @@
+#ifndef PHOEBUS_UTILS_PREPROCESSOR_UTILS_HPP_
+#define PHOEBUS_UTILS_PREPROCESSOR_UTILS_HPP_
+
+#define TOKEN2STRING_IMPL(x) #x
+#define TOKEN2STRING(x) TOKEN2STRING_IMPL(x)
```

Comment:
Preprocessor tokenization. I have never understood why you need both of these, and can't just do `#x`. But whatever.